.ado/jobs/cli-init.yml

```diff
@@ -4,273 +4,265 @@ parameters:
     default : PullRequest
```

Comment:
The only changes here are:
* Extra enum value in the parameter
* An extra `${{ if` above each job to separate the nuget from non nuget
* Removal of the dependsOn and Conditions of the jobs

GitHub's diff is doing a pretty poor job dealing with the indentation...
Opening the PR in codeflow shows a way better diff.